app/_redirects

```diff
@@ -133,6 +133,8 @@
 /konnect/servicehub/plugins/enable-service-plugin/  /konnect/manage-plugins/enable-service-plugin/
 /konnect/servicehub/plugins/enable-route-plugin/    /konnect/manage-plugins/enable-route-plugin/
 /konnect/dev-portal/administrators/customization/themes         /konnect/dev-portal
+/konnect/migrate                                    /konnect/deployment/import
```

Comment:
Should this be `/konnect/deployment/migrate`?

Reply:
I understand - I meant the old URL. I've checked and it is `/konnect/migrate` not `/konnect/deployment/migrate` in prod 👍